test/playwright/e2e/report-media.spec.ts

```diff
@@ -0,0 +1,107 @@
+import { test, expect, Page, BrowserContext } from '@playwright/test'
+
+import { mockProviderApis } from '~~/test/playwright/utils/route'
+
+/**
+ * Some helpers for repeated actions.
+ */
+
+export const visitFirstResult = (page: Page) =>
+  page.click('[data-testid="search-results"] a:first-child')
+export const openReportModal = (page: Page) =>
+  page.click('text="Report this content"')
+
+const submitDmcaReport = async (page: Page, context: BrowserContext) => {
+  // Mock the Google Form to return a successful html document
+  await context.route('https://docs.google.com/forms/**', (route) => {
+    route.fulfill({
+      status: 200,
+      contentType: 'text/html',
+      body: '<div>Fake form!</div>',
+    })
+  })
+  await page.click('text="Infringes copyright"')
+  const [newPage] = await Promise.all([
+    context.waitForEvent('page'),
+    await page.click('text="Open form"'), // Opens a new tab
+  ])
+  await newPage.waitForLoadState()
+  return expect(await newPage.url()).toContain('https://docs.google.com/forms')
+}
+
+// todo: Test a mature report with the optional description field
+const submitMatureContentReport = async (
+  page: Page,
+  context: BrowserContext
+) => {
+  const reportingEndpoint = /https:\/\/api.openverse.engineering\/\w+\/\w+/
```

Comment:
Oops, need to fix this. The whole reason I used a regex in the first place was to match `https://api.openverse.engineering/{mediatype}/{id}/report/`